README.md

```diff
@@ -194,15 +194,15 @@ The actions to wait for. It can be either:
 
 #### timeout
 
-Type: `Number`   
+Type: `Number`
 Default: 50
 
 The timeout given in milliseconds.
 
 
 ## Tests
 
-`$ npm test`   
+`$ npm test`
```

Comment:
Please add the spaces back because that is how you insert a new line in markdown.